interfaces/Portalicious/src/app/services/auth/auth-strategy.interface.ts

```diff
@@ -1,13 +1,16 @@
-import { Type } from '@angular/core';
+import { EnvironmentProviders, Provider, Type } from '@angular/core';
 
 import { User } from '~/domains/user/user.model';
-import { LocalStorageUser } from '~/services/auth.service';
+import { LocalStorageUser } from '~/utils/local-storage';
 
 export abstract class IAuthStrategy {
+  public static readonly APP_PROVIDERS: (EnvironmentProviders | Provider)[];
+
   LoginComponent: Type<unknown>;
   ChangePasswordComponent: null | Type<unknown>;
+  CallbackComponent: null | Type<unknown>;
```

Comment:
```suggestion
  CallbackComponent: Type<unknown>;
```

Maybe... (see my other comments)